src/main/java/org/gridsuite/modification/modifications/AbstractBranchModification.java

```diff
@@ -190,26 +195,32 @@ private void detectApplicabilityChange(Branch<?> branch, List<OperationalLimitsG
         }
     }
 
-    private void modifyOperationalLimitsGroups(Branch<?> branch, List<OperationalLimitsGroupModificationInfos> operationalLimitsInfos, List<ReportNode> olgReports) {
+    private void modifyOperationalLimitsGroups(Branch<?> branch, List<OperationalLimitsGroupModificationInfos> operationalLimitsInfos,
+                                               ReportNode limitSetsReportNode) {
+
         for (OperationalLimitsGroupModificationInfos opLGModifInfos : operationalLimitsInfos) {
             if (opLGModifInfos.getModificationType() == null) {
                 continue;
             }
+
+            ArrayList<ReportNode> limitSetReports = new ArrayList<>();
```

Comment:
I know the users still use 'limit sets' (and personally I prefer limits sets) but I think we should only use one term in the code, and I think this will have to be powsybl's term : operational limits groups
```suggestion
            ArrayList<ReportNode> olgReports = new ArrayList<>();
```

Reply:
if you wish